dev/io.openliberty.mpTelemetry.2.1.thirdparty/bnd.bnd

```diff
@@ -0,0 +1,49 @@
+#*******************************************************************************
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+publish.wlp.jar.suffix: dev/api/third-party
```

Comment:
```suggestion
publish.wlp.jar.suffix: dev/api/stable
```
I believe this would have a different name as well like `io.openliberty.io.opentelemetry.2.1` which already exists and if it has exactly the same content as this bundle, then we can just put that bundle in `dev/api/stable` and not have it in lib

Reply:
More to happen here for using io.openliberty.io.opentelemetry in `stable` dir instead of this thirdparty bundle name?

Reply:
The content is different. `io.openliberty.io.opentelemetry.2.1` is what pulls in the OpenTelemetry dependencies that we don't want to expose to users. I think `io.openliberty.mpTelemetry.2.1.api` would be better.

Reply:
The [naming guidelines](https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-naming-guidelines) do not include a `.api` option for APIs.  So it would seem that you would want to have `io.openliberty.io.opentelementry.2.1` in the stable directory and maybe the other jar renamed to something else.

Reply:
Almost there.  The third party bundle still has the wrong name.  You updated the feature file to reference the new name of `io.openliberty.io.opentelemetry.2.1`, but that bundles doesn't exist.  Only the `io.openliberty.mpTelemetry.2.1.api` one does.  So this third party project should be renamed to `io.openliberty.io.opentelemetry.2.1` and the symbolic name updated.